### ISSUE TITLE
Volume timepoints are not working anymore

### ISSUE THREAD

**Issue by kephale** [scenerygraphics/sciview#309]
If you run this script and then try to `Play` the time series of the volume via the inspector, it does not play.

```
# @SciView sv

import net.imglib2.img.array.ArrayImgs as ArrayImgs
import bdv.util.BdvFunctions as BdvFunctions

w = 50
h = w
d = w
tmax = 10

img = ArrayImgs.unsignedBytes(w, h, d, tmax)

x = 0
y = 0
z = 0
t = 0

c = img.cursor()
while c.hasNext():
    c.fwd()
    x = c.getFloatPosition(0)
    y = c.getFloatPosition(1)
    z = c.getFloatPosition(2)
    t = c.getFloatPosition(3)

    c.get().setReal(((x%15 + y%15 + z%15) * (t%4))%255)

BdvFunctions.show(img, "test")

sv.addVolume(img)

```

**Comment by kephale** [comment]
Not quite true. It is actually the same problem as https://github.com/scenerygraphics/sciview/issues/308

If you disable push mode, then it works again.

**Comment by skalarproduktraum** [comment]
This issue is fixed in #304.
